soccer/src/soccer/ui/main_window.cpp

```diff
@@ -1138,19 +1138,19 @@ void MainWindow::on_fastKickoffYellow_clicked() {
 }
 
 void MainWindow::on_fastDirectBlue_clicked() {
```

Comment:
can you remove the direct/indirect buttons in the main window UI and replace with a free kick button? this could be confusing to someone in the future if there are direct/indirect buttons but they do the same thing